pontoon/base/models.py

```diff
@@ -2316,6 +2316,25 @@ def rejected(self, locale, project=None):
             )
         )
 
+    def missing_no_suggestions(self, locale, project=None):
+        """Return a filter to be used to select entities with no or only rejected translations.
+
+        This filter will return all entities that have no or only rejected translations.
+        :arg Locale locale: a Locale object to get translations for
+
+        :returns: a django ORM Q object to use as a filter
+
+        """
+        return ~Q(
+            pk__in=self.get_filtered_entities(
+                locale,
+                Q(approved=True) | Q(rejected=False),
```

Comment:
We also need to exclude Fuzzies.

Reply:
Fuzzies I think will fall in one of the two categories. Can you please give me an example where this is the case?  Rejected=False is good enough to check the presence of Fuzzies as well

Reply:
You're right again! Could you please add a comment about `rejected=False` also catching `fuzzy=True`? I got confused when comparing with the `missing()` filter, maybe somebody else will, too.

Reply:
I still added fuzzies to make it easy to understand. Let me know if we want to change it